src/wallet/crypter.h

```diff
@@ -185,6 +187,34 @@ class CCryptoKeyStore : public CBasicKeyStore
             mi++;
         }
     }
+    virtual bool AddCryptedSpendingKey(const libzcash::PaymentAddress &address, const std::vector<unsigned char> &vchCryptedSecret);
+    bool AddSpendingKeyPaymentAddress(const libzcash::SpendingKey &key, const libzcash::PaymentAddress &address);
+    bool HaveSpendingKey(const libzcash::PaymentAddress &address) const
+    {
```

Comment:
I thought that might be it. I don't think I'll ever work out how C++ programmers decide this :-/